src/api/java/io/github/cvc5/Utils.java

```diff
@@ -139,25 +149,25 @@ public static void loadLibraries()
     {
       try
       {
-        System.loadLibrary("cvc5jni");
+        // Try to extract the libraries from a JAR in the classpath
+        List<String> filenames = readLibraryFilenames(LIBPATH_IN_JAR + "/filenames.txt");
```

Comment:
Will "/..." work for all supported platforms or should be File.separator used instead?

Reply:
It works on all supported platforms because it is a resource path, not a file system path.